google-common/src/main/scala/akka/stream/alpakka/google/auth/JwtSprayJsonParser.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+/*
+ * This software is licensed under the Apache 2 license.
+ * Copyright 2021 JWT-Scala Contributors.
```

Comment:
can the copyright be placed this at the top header?

```
/*
 * Copyright 2021 JWT-Scala Contributors.
 * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
 */
```

(and it's not really since 2016)

Reply:
That would require excluding this file from the sbt-header plugin.

Reply:
But not too hard.